resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -891,6 +897,37 @@ class FilterKeys {
     }
 }
 
+[DscResource()]
+class EyeControl {
+    [DscProperty(Key)] [Ensure] $Ensure

```

Comment:
Can we move forward with the `$Exist` property for DSC versions and higher?

Reply:
Boolean values can’t be used as a key